app/service/users_report_service.rb

```diff
@@ -0,0 +1,34 @@
+class UsersReportService
+  def user_data
+    {
+      caption: I18n.t("reports.users.heading"),
+      head: [
+        { text: I18n.t("reports.users.table_headings.organisation_name") },
+        { text: I18n.t("reports.users.table_headings.user_count"), numeric: true },
+      ],
+      rows:,
```

Comment:
I think it would make sense to make the first cell a row header, as in the design system examples: https://design-system.service.gov.uk/components/table/

```suggestion
      rows:,
      first_cell_is_header: true
```